src/cli.ts

```diff
@@ -30,7 +30,11 @@ console.info(`Output directory: ${params.out}`);
   const proxy = (await getProxySettings()) as ProxySettings | null; // TODO: remove `as ...` when https://github.com/Azure/get-proxy-settings/issues/24 is fixed
   const bestProxy = (proxy && (proxy.https || proxy.http)) || undefined; // TODO: remove `proxy && ` when https://github.com/Azure/get-proxy-settings/issues/24 is fixed
 
-  const app = new App(params.out, bestProxy);
+  const app = new App({
+    discoveryJsonDirectory: process.env.GAPI_DISCOVERY_JSON_DIR,
```

Comment:
Thoughts on this approach? Perhaps this should be a CLI option instead?

Reply:
I think that `discoveryJsonDirectory` should be CLI arg indeed. This way we can easily add some helping text for running generator, which can be helpful if anyone not familiar with the codebase will need to debug it locally.